nomenclature/definitions/variable/industry_types.yaml

```diff
@@ -0,0 +1,122 @@
+# This list of industries will replace any occurrence of `<industry>`
+# in a variable-definition yaml file
+
+
+#   i.e., The following standard generical structure in economic variables:
+#   CapitalPriceIndex|<industry>:
+#   LaborPriceIndex|<industry>:
+
+<Industry>:
+  iPLNT:
```

Comment:
This is not intuitive to non-experts, any chance to use slightly more descriptive names?

Reply:
@hettieboonman, could you bring us an additional comment to rename the `iPLNT` industry -type, please?

Reply:
Irrelevant per updated list of industries